packages/deprecation-crawler/src/lib/output-formatters/add-comments-to-repository.ts

```diff
@@ -5,11 +5,6 @@ export async function addCommentToRepository(
   config: CrawlConfig,
   rawDeprecations: Deprecation[]
 ): Promise<void> {
-  if (rawDeprecations.length === 0) {
```

Comment:
Removed all these checks, these are now handled in the precondition (and it doesn't log anything).